src/net_processing.cpp

```diff
@@ -4243,12 +4279,17 @@ bool PeerManagerImpl::SendMessages(CNode* pto)
                     }
                 }
             }
+
             pto->vAddrToSend.clear();
-            if (!vAddr.empty())
+
+            if (!vAddr.empty()) {
```

Comment:
I suggest you drop these tidy-up changes in _Add some documentation and small style fixes_. They don't seem very related to the rest of the PR :see_no_evil:

Reply:
sorry I'm not sorry!! I abhor two line if statements, they are great footguns. these changes are pretty trivial, I don't think they would make sense as a stand-alone PR, or that they increase the complexity of review, so I'd rather leave them in.

Reply:
I agree that two line if statements are undesirable. If you were already touching these lines, then I'd agree that you should improve them. However, these are the only changes you make to `SendMessages()` and are unrelated to the rest of the changes in this PR.

I refactor this entire chunk of logic in #21236. Without this tidy-up commit, the two PRs don't conflict and can be merged in either order. If you include these changes, then the PRs conflict and one will have to be rebased when the other is merged.

Reply:
ok! reverted :)